src/iotjs_binding.h

```diff
@@ -293,6 +295,24 @@ static inline bool ge(uint16_t a, uint16_t b) {
 
 #define JHANDLER_FUNCTION(name) static void name(iotjs_jhandler_t* jhandler)
 
+#define JHANDLER_CHECK_NATIVE_INFO(x, native_info)               \
+  if (!iotjs_jval_get_object_native_handle_s(x, &native_info)) { \
+    JHANDLER_THROW(COMMON, "Unsafe access");                     \
+    return;                                                      \
+  }
+
+#define JHANDLER_CHECK_THIS_SAFE(type) \
+  JHANDLER_CHECK_NATIVE_INFO(JHANDLER_GET_THIS(type), this_module_native_info)
+
+#define JHANDLER_DECLARE_WRAPPER(type, name)                           \
+  JHANDLER_CHECK_THIS(object);                                         \
+  iotjs_##type##_t* name = (iotjs_##type##_t*)                         \
+      iotjs_jval_get_object_native_handle_s(JHANDLER_GET_THIS(object), \
```

Comment:
I would do the object check (`JHANDLER_CHECK_THIS(object)`) inside `iotjs_jval_get_object_native_handle_s`. This would reduce binary size since this is a frequently used function. Whit is the _s postfix? static?